### ISSUE TITLE
how about results(mIoU) on validation set and test set

### ISSUE THREAD

**Issue by jwnirvana** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#62]
could you please provide results(mIoU) on validation set and test set?
Thanks

**Comment by zhengyang-wang** [comment]
The results on validation sets are provided in the paper. We didn't upload results to the testing server.

**Comment by jwnirvana** [comment]
thanks
